terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/main/java/org/terasoluna/gfw/web/el/Functions.java

```diff
@@ -166,7 +166,8 @@ public static String u(String value) {
         if (value == null || value.isEmpty()) {
             return "";
         }
-        return UriUtils.encodeQueryParam(value, "UTF-8");
+        // replace "+" -> "%2B" due to Spring Framework 5.x specification change.
+        return UriUtils.encodeQueryParam(value, "UTF-8").replace("+", "%2B");
```

Comment:
Agree.
However, since the encoding target is "Query" not "Query Param", the method name is `extraEncodeQuery`.
In Spring 4.x and earlier, "+" in the key of query is also percent encoded.

d2e18ce9b215aa7feddb1283a06589bd6bc722a5